### DISCUSSION TITLE
Webtop rename and open

### DISCUSSION THREAD

**Discussion by UlrichThiess** [alexbelgium/hassio-addons#1571] [Ideas]
Why not change the name webtop_kde to webtop? => shorter, easier to type, kde is simply wrong

Second: Linuxserver.io offers many types, as XFCE or KDE, ubuntu or debian and so on. Could it be possible to make this configurable?

**Comment by alexbelgium** [comment]
Hi, I'll change the name indeed. Regarding configuration it is not possible as linked to which upstream image is used ; also not all images can be used with docker without admin rights. I had selected this one as fitting, but others did not easily. However it is super easy to fork the repo and modify it if you have a specific usage

**Comment by UlrichThiess** [comment]
Thanks. Not a specific usage, just playing around.
